### ISSUE TITLE
missing parameters in prod.yaml file

### ISSUE THREAD

**Issue by bintocher** [ptarmiganlabs/butler-sos#602]
### What version of Butler SOS are you using?

docker-compose up

### What command did you use to start Butler SOS?

docker-compose up

### What operating system are you using?

Docker

### What CPU architecture are you using?

x86_64

### What Qlik Sense versions are you using?

2023-feb sr2

### Describe the Bug

file https://github.com/ptarmiganlabs/butler-sos/blob/master/src/config/production_template.yaml
have missing params to run 9.6.1 version in newRelic section:
```
  newRelic:
    metric:
      dynamic:
        apps:
          activeDocs:
            enable: true
          loadedDocs:
            enable: true
          inMemoryDocs:
            enable: true
```
without this params i have errors like this:

```
butler-sos  | 2023-08-22T10:05:57.992Z error: /nodeapp/config/production_influxdb.yaml is not following the correct structure, missing:,Butler-SOS.newRelic.metric.dynamic.apps.activeDocs.enable
butler-sos  | 2023-08-22T10:05:59.942Z error: /nodeapp/config/production_influxdb.yaml is not following the correct structure, missing:,Butler-SOS.newRelic.metric.dynamic.apps.loadedDocs.enable
butler-sos  | 2023-08-22T10:06:15.192Z error: /nodeapp/config/production_influxdb.yaml is not following the correct structure, missing:,Butler-SOS.newRelic.metric.dynamic.apps.inMemoryDocs.enable
```

**Comment by mountaindude** [comment]
Acknowledged. 
The template file has a few missing parameters.

Should be fixed in version 9.6.2 which was released a few minutes ago.
Docker image is building right now, done in a few minutes.

**Comment by mountaindude** [comment]
9.6.2 Docker image is now on Docker Hub.
